### PR TITLE
Fix AWS Lambda advanced docs

### PR DESCRIPTION
**Pull request by mattn**

### CODE REVIEW

**Comment by cpanato** [comment]
please sign the DCO

thanks for the PR!

File: docs/advanced/lambda.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ type Event struct {
 func main() {
     lambda.Start(func(ctx context.Context, event Event) (string, error) {
         return fmt.Sprintf("Hello %s!", event.Name), nil
-    }
+    })
```
Comment (cpanato) [inline docs/advanced/lambda.md:26]: actually the correct is

```
func main() {
	lambda.Start(func(ctx context.Context, event Event) (string, error) {
		return fmt.Sprintf("Hello %s!", event.Name), nil
	})
}
```

**Review by cpanato — CHANGES_REQUESTED** [review]

File: docs/advanced/lambda.md (hunk 1 above)
Comment (mattn) [reply docs/advanced/lambda.md:26]: What do you mean? I did fix it already. Do you mean replacing 4 spaces into tabs?

**Review by imjasonh — APPROVED** [review]

File: docs/advanced/lambda.md (hunk 1 above)
Comment (imjasonh) [reply docs/advanced/lambda.md]: I don't feel strongly about the tabs vs spaces, folks who use the code will have it gofmt'ed.

**Review by cpanato — APPROVED** [review]

**Comment by mattn** [comment]
Ooops, race condition of commit & you comment. :(
I updated the Go code. (replaced 4 spaces into tabs)

**Comment by mattn** [comment]
BTW, I'm trying ko with AWS Lambda, but it seems that `ko build` require `--bare` for AWS ECR. The docs is correctly?

**Pull request merged**